org-cyf-itp/content/onboarding/success/index.md

```diff
@@ -0,0 +1,32 @@
++++
+title = 'success'
+description = 'How do we know we have completed User Focused Data?'
+layout = 'success'
+emoji= '✅'
+menu_level = ['module']
+weight = 11
+[[objectives]]
+1="Link to your PR for [Wireframe to Web Code](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/90)"
+2="Link to your PR for [Form Controls](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/88)"
+3="Link to your Google Sheet for [Query Data in Google Sheets with SQL](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/104)"
```

Comment:
For the 3rd objective - could we ask them to link to their Google sheet for this coursework instead: [SQL Problems using Google Sheets](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/103)?
The currently linked coursework seems like it's more of a video / code-along, where as the one I'm suggesting would hopefully give us a better idea of their understanding.

Reply:
As per our call, think for step 1 let's just ask for the 2 pieces of coursework and drop everything else